dockerfiles/build/www/wp-plugins/local.txt

```diff
@@ -0,0 +1,9 @@
+# List of zipped WordPress plugins to unzip from a local directory.
```

Comment:
The local directory as in the root of the repository, or some other specific subdirectory predefined/supplied by the docker build? Looks like it can be any supplied dir based on the unzip script, but just making sure.